packages/device_info/android/src/main/java/io/flutter/plugins/deviceinfo/MethodCallHandlerImpl.java

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.deviceinfo;
+
+import android.annotation.SuppressLint;
+import android.content.ContentResolver;
+import android.os.Build;
+import android.provider.Settings;
+import androidx.annotation.NonNull;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The implementation of {@link MethodChannel.MethodCallHandler} for the plugin. Responsible for
+ * receiving method calls from method channel.
+ */
+public class MethodCallHandlerImpl implements MethodChannel.MethodCallHandler {
```

Comment:
Can we make this class default (package-private) so that we don't need to worry about breaking other clients who may be using it?

Reply:
done